### PR TITLE
Release notes for June 30

### PR DESCRIPTION
**Pull request by srpearce**
@lamagnifica I'm still waiting on URLs for the hinting docs, but wanted to go ahead and create this PR now. Feel free to review.

### CODE REVIEW

**Comment by lamagnifica** [comment]
@srpearce are these the release notes? I'm confused.

**Comment by srpearce** [comment]
@lamagnifica Yep - sorry, thought the other files were included in the commit and didn't verify. :-|

File: en_us/release_notes/source/2015/documentation/doc_0630_2015.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Building and Running an edX Course
+**********************************
+
+* A new `Processing Video Files_partners`_ section describes the workflow for
+  creating videos and uploading them for use in an edx.org course.
+
+* A `Name_partners`_ section describes the new problem templates that include
+  hinting and feedback.
+
```
Comment (lamagnifica) [inline en_us/release_notes/source/2015/documentation/doc_0630_2015.rst]: were these two links added to the links.rst file?

File: en_us/release_notes/source/2015/documentation/doc_0630_2015.rst [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+==================================
+Documentation for Open edX Users
+==================================
+
+Building and Running an Open edX Course
+****************************************
+
+* A new `Processing Video Files_openedx`_ section describes the workflow for
```
Comment (lamagnifica) [inline en_us/release_notes/source/2015/documentation/doc_0630_2015.rst]: strike this bullet. This section should NOT be in the open edX version of the guide. The process applies ONLY to videos in edx.org courses!

File: en_us/release_notes/source/2015/documentation/doc_0630_2015.rst [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+Documentation for edX Partners
+==================================
+
+Building and Running an edX Course
+**********************************
+
+* A new `Processing Video Files_partners`_ section describes the workflow for
+  creating videos and uploading them for use in an edx.org course.
+
+* A `Name_partners`_ section describes the new problem templates that include
```
Comment (lamagnifica) [inline en_us/release_notes/source/2015/documentation/doc_0630_2015.rst]: @mhoeber are hints and feedback openedx, partners, or both?

File: en_us/release_notes/source/2015/studio/studio_0630_2015.rst [hunk 4]
```diff
@@ -7,12 +7,12 @@ You can now add feedback, hints, or both to the following problem types.
 
 * Checkbox
 * Dropdown
-* Multiple Choice
-* Numerical Input
-* Text Input
+* Multiple choice
+* Numerical input
+* Text input
```
Comment (lamagnifica) [inline en_us/release_notes/source/2015/studio/studio_0630_2015.rst]: the names of the problem types are icaps, not lower case. If the names changed in the UI, refernces to them will need to be changed in a lot more places than this... please make a story if so.

File: en_us/release_notes/source/2015/documentation/doc_0630_2015.rst (hunk 3 above)
Comment (mhoeber) [inline en_us/release_notes/source/2015/documentation/doc_0630_2015.rst]: both

File: en_us/release_notes/source/2015/documentation/doc_0630_2015.rst (hunk 1 above)
Comment (srpearce) [inline en_us/release_notes/source/2015/documentation/doc_0630_2015.rst]: Waiting on URLs from @mhoeber

File: en_us/release_notes/source/2015/studio/studio_0630_2015.rst (hunk 4 above)
Comment (srpearce) [inline en_us/release_notes/source/2015/studio/studio_0630_2015.rst]: The UI selections are icapped, but our documentation doesn't cap them in descriptions (e.g., "In numerical input problems, learners enter numbers..."; "For the multiple choice problems in your course..."). I don't think this list here needs to reflect the UI.

Comment (lamagnifica) [inline en_us/release_notes/source/2015/studio/studio_0630_2015.rst]: @srpearce how does that square with "Instructor Dashboard", icaps, not bold (for example)?

Comment (srpearce) [inline en_us/release_notes/source/2015/studio/studio_0630_2015.rst]: We've been using "Instructor Dashboard" as a proper noun all along, though I don't think we ever discussed it (I've just been following what's already there - and there was a tiny bit of discussion at one point over whether we should use the same convention for the learner dashboard, and we should probably figure that out). To me, problem types aren't proper nouns, just like component types aren't proper nouns (though we used to treat them like proper nouns, which is my fault!).

File: en_us/release_notes/source/2015/documentation/doc_0701_2015.rst [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+==================================
+
+Building and Running an edX Course
+**********************************
+
+* A new `Processing Video Files`_ section describes the workflow for
+  creating videos and uploading them for use in an edx.org course.
+
+* A `Name_partners`_ section describes the new problem templates that include
+  hinting and feedback.
```
Comment (srpearce) [inline en_us/release_notes/source/2015/documentation/doc_0701_2015.rst]: Note: Waiting on URL

File: en_us/release_notes/source/2015/documentation/doc_0701_2015.rst [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+==================================
+Documentation for Open edX Users
+==================================
+
+Building and Running an Open edX Course
+****************************************
+
+* A `Name_openedx`_ section describes the new problem templates that include
+  hinting and feedback.
```
Comment (srpearce) [inline en_us/release_notes/source/2015/documentation/doc_0701_2015.rst]: Note: Waiting on URL

File: en_us/release_notes/source/2015/lms/lms_0701_2015.rst [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+For example, when the course team configures hints, learners can view those hints by
+selecting **Hint** under the problem.
+
+.. image:: /Images/multiple_choice_hint.png
+  :alt: Hints in a multiple choice problem.
+  :width: 500
+
+
+========================================
```
Comment (srpearce) [inline en_us/release_notes/source/2015/lms/lms_0701_2015.rst]: @marcotuts Can you let me know if you're OK with the following text re: capa styling changes?

File: en_us/release_notes/source/2015/documentation/doc_0701_2015.rst (hunk 6 above)
Comment (lamagnifica) [inline en_us/release_notes/source/2015/documentation/doc_0701_2015.rst]: no need to bullet if there's only one item in a list

File: en_us/release_notes/source/2015/lms/lms_0701_2015.rst [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+To make the edX LMS easier to use on mobile devices, the appearance of common
+problem types has changed. For example, a border surrounds options for
+multiple choice and checkbox problems, making it easier for learners to select
+an option.
+
+.. image:: /Images/new_capa_styling.png
+  :alt: Checkbox problem showing the new problem styles.
+  :width: 500
+
+Common problem types include the following types.
```
Comment (lamagnifica) [inline en_us/release_notes/source/2015/lms/lms_0701_2015.rst]: The problem types affected by this change follow.

File: en_us/release_notes/source/2015/lms/lms_0701_2015.rst [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+
+* Checkbox
+* Dropdown
+* Multiple choice
+* Numerical input
+* Text input
+
+
```
Comment (lamagnifica) [inline en_us/release_notes/source/2015/lms/lms_0701_2015.rst:47]: possibly, you might reorg to say that there are two changes in this release that affect this set of problem types. 1) hints & feedback 2) styling...

**Pull request merged**